src/cephadm/tox.ini

```diff
@@ -62,7 +62,7 @@ deps =
     flake8-quotes
 commands =
     flake8 --config=tox.ini {posargs:cephadm.py cephadmlib}
-    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | xargs grep "docker.io" | wc -l) == 11'
+    bash -c 'test $(git ls-files 'cephadm.py' 'cephadmlib/*.py' | sort -u | xargs grep "docker.io" | wc -l) == 11'
```

Comment:
Just curios about what are we trying to achieve here. The output of this command has 11 entries but they include some entries from comments/code which doesn't seem a "stable" to rely on!! 

```
cephadmlib/constants.py:DEFAULT_LOKI_IMAGE = 'docker.io/grafana/loki:2.4.0'
cephadmlib/constants.py:DEFAULT_PROMTAIL_IMAGE = 'docker.io/grafana/promtail:2.4.0'
cephadmlib/constants.py:DEFAULT_SNMP_GATEWAY_IMAGE = 'docker.io/maxwo/snmp-notifier:v1.2.1'
cephadmlib/constants.py:DEFAULT_REGISTRY = 'docker.io'  # normalize unqualified digests to this
cephadmlib/packagers.py:            self.install(['docker.io'])
cephadm.py:    >>> normalize_image_digest('ceph/ceph', 'docker.io')
cephadm.py:    'docker.io/ceph/ceph'
cephadm.py:    >>> normalize_image_digest('quay.ceph.io/ceph/ceph', 'docker.io')
cephadm.py:    >>> normalize_image_digest('docker.io/ubuntu', 'docker.io')
cephadm.py:    'docker.io/ubuntu'
cephadm.py:    >>> normalize_image_digest('localhost/ceph', 'docker.io')
```
In case we are trying to count the number of container images then I think we should restrict the search to the file **cephadmlib/constants.py**, WDYT?

Reply:
This thing that counts the docker.io instances has been around a really long time, and was intended to be useful for downstream versions to be aware of the number of references to docker.io in the codebase. I think this change is only to make it work properly in weird temporary git situations (the description mentions rebases and merges) that cause it to list the same file multiple times. There's no intention to actually change any behavior here, just make what's there work in those states.

Reply:
Yeah, in an ideal world it would only need to look at constants.py, but since we can't prevent people from doing things like:
```python
def silly_function(foo, bar, default_image='docker.io/blat/frob:xyz'):
    ...
```
It's best to let the check cover all the source files for now. Initially, I wanted to get rid of the check entirely. But I was talked out of that. :-)

Reply:
Oh BTW, I'm pretty sure that even explicitly listing cephadmlib/constants.py would not necessarily prevent `git ls-files` from printing duplicates. It's just one of those weird un-ergonomic things git likes to do IMO.